config/post-install/storage-version-migrator.yaml

```diff
@@ -55,6 +55,7 @@ spec:
             - "subscriptions.messaging.knative.dev"
             - "triggers.eventing.knative.dev"
             - "jobsinks.sinks.knative.dev"
+            - "eventpolicies.eventing.knative.dev"
```

Comment:
Just to be sure, as I haven't worked with the storage-verion-migrator so far: this does not fail, when we only have one version available so far?

Reply:
there is a noop if no changes are.  For most of the others we also only have on version, e.g. `v1`

Reply:
This reminds me of doing the same for the `IntegrationSource/sink` bits :sweat_smile: